netmiko/base_connection.py

```diff
@@ -1028,6 +1028,8 @@ def send_command(self, command_string, expect_string=None,
         while i <= max_loops:
             new_data = self.read_channel()
             if new_data:
+                if self.ansi_escape_codes:
+                    new_data = self.strip_ansi_escape_codes(new_data)
```

Comment:
I am a bit worried about how slow this is going to be...I assume this was on some Linux platform (I seem to recall on Slack or on your GitHub issue that you had stated it was)?

Reply:
I have to use Windows 10 for my development machine.

Reply:
This is probably fine...let me just see how long this takes to execute, but it is probably no issue.